applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -3,8 +3,11 @@
 Application description
 #######################
 
-The Serial LTE Modem (SLM) application demonstrates how to use the nRF9160 as a stand-alone LTE modem that can be controlled by proprietary AT commands.
+.. contents::
+   :local:
+   :depth: 2
```

Comment:
```suggestion
   :depth: 3
```
That would make it more usable - but I don't know if we want to be consistent with the level here.

Reply:
As just discussed, we will fix this in a different PR.